playbook/app/pb_kits/playbook/pb_file_upload/_file_upload.scss

```diff
@@ -3,4 +3,22 @@
     border: 1px #ccc dashed;
     text-align: center;
   }
+  // Overrides default focus styles only for File Upload
+  .text_input:focus {
+    border: 1px solid $border_light !important;
+    background-color: transparent !important;
+  }
+  ::file-selector-button {
+    display: none;
+  }
+  &.full_width {
+    [class*='pb_form_group_kit'] {
+      width: 100% !important;
+      
+      [class*='pb_text_input_kit'] {
+        width: 100% !important;
+      }
+    }
+  }
```

Comment:
Since your using a form group can you not just pass the full prop to the group rather than custom CSS. You can see the css for that grabs all divs: https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/pb_form_group/_form_group.scss#L7-L13